composer.json

```diff
@@ -31,7 +31,8 @@
         "swiftmailer/swiftmailer": "~4.3|~5",
         "willdurand/propel-typehintable-behavior": "~1.0",
         "symfony/yaml": "~2.3",
-        "symfony/validator": "~2.3"
+        "symfony/validator": "~2.3",
+        "symfony/phpunit-bridge": "2.8.x-dev"
```

Comment:
please don't force using the dev versions here. I still want to be able to run things against stable deps